app/Models/BooleanSearch.php

```diff
@@ -74,7 +74,11 @@ private function parseUserQueryNames(string $input): string {
 					$name = trim($matches['search'][$i]);
 					if (!empty($queries[$name])) {
 						$fromS[] = $matches[0][$i];
-						$toS[] = '(' . trim($queries[$name]->getSearch()->getRawInput()) . ')';
+						if ($allowUserQueries) {
+							$toS[] = '(' . trim($queries[$name]->getSearch()->getRawInput()) . ')';
+						} else {
+							$toS[] = '';
+						}
```

Comment:
```suggestion
						$toS[] = '';
						if ($allowUserQueries) {
							$toS[] = '(' . trim($queries[$name]->getSearch()->getRawInput()) . ')';
						}
```

Reply:
Reduce cyclomatic complexity

Reply:
Sorry, that is wrong (you would happen twice...)

Reply:
Sorry, you’r right